troubleshoot/ports/troubleshoot_ports.go

```diff
@@ -0,0 +1,34 @@
+package ports
+
+import (
+	"fmt"
+	"strings"
+)
+
+func TroubleshootDefaultPorts(host string) {
+	// Source - https://developer.hashicorp.com/consul/docs/install/ports
+	ports := []string{"8600", "8500", "8501", "8502", "8503", "8301", "8302", "8300"}
+	TroubleshootRun(ports, host)
+}
+
+func TroubleShootCustomPorts(host string, ports string) {
+	portsArr := strings.Split(ports, ",")
+	TroubleshootRun(portsArr, host)
+}
+
+func TroubleshootRun(ports []string, host string) {
+
+	resultsChannel := make(chan string)
+
+	var counter = 0
```

Comment:
goland says it can be ommited.
![Screenshot 2023-08-28 at 1 55 50 PM](https://github.com/hashicorp/consul/assets/134911583/ffa1b557-f95f-430a-8a00-ec63e3170fb3)

Reply:
You can remove `= 0`. `var counter int` should take care of the zero valued initialization.